examples/Migration/CampaignManagement/CreateCompleteCampaignBothApisPhase1.php

```diff
@@ -19,10 +19,427 @@
+use Google\Ads\GoogleAds\V1\Enums\BudgetDeliveryMethodEnum\BudgetDeliveryMethod;
+use Google\Ads\GoogleAds\V1\Resources\CampaignBudget;
+use Google\Ads\GoogleAds\V1\Services\CampaignBudgetOperation;
+use Google\AdsApi\AdWords\AdWordsServices;
+use Google\AdsApi\AdWords\AdWordsSession;
+use Google\AdsApi\AdWords\v201809\cm\AdGroup;
+use Google\AdsApi\AdWords\v201809\cm\AdGroupAd;
+use Google\AdsApi\AdWords\v201809\cm\AdGroupAdOperation;
+use Google\AdsApi\AdWords\v201809\cm\AdGroupAdRotationMode;
+use Google\AdsApi\AdWords\v201809\cm\AdGroupAdService;
+use Google\AdsApi\AdWords\v201809\cm\AdGroupAdStatus;
+use Google\AdsApi\AdWords\v201809\cm\AdGroupCriterion;
+use Google\AdsApi\AdWords\v201809\cm\AdGroupCriterionOperation;
+use Google\AdsApi\AdWords\v201809\cm\AdGroupCriterionService;
+use Google\AdsApi\AdWords\v201809\cm\AdGroupOperation;
+use Google\AdsApi\AdWords\v201809\cm\AdGroupService;
+use Google\AdsApi\AdWords\v201809\cm\AdGroupStatus;
+use Google\AdsApi\AdWords\v201809\cm\AdRotationMode;
+use Google\AdsApi\AdWords\v201809\cm\AdvertisingChannelType;
+use Google\AdsApi\AdWords\v201809\cm\BiddableAdGroupCriterion;
+use Google\AdsApi\AdWords\v201809\cm\BiddingStrategyConfiguration;
+use Google\AdsApi\AdWords\v201809\cm\BiddingStrategyType;
+use Google\AdsApi\AdWords\v201809\cm\Budget;
+use Google\AdsApi\AdWords\v201809\cm\Campaign;
+use Google\AdsApi\AdWords\v201809\cm\CampaignOperation;
+use Google\AdsApi\AdWords\v201809\cm\CampaignService;
+use Google\AdsApi\AdWords\v201809\cm\CampaignStatus;
+use Google\AdsApi\AdWords\v201809\cm\CpcBid;
+use Google\AdsApi\AdWords\v201809\cm\ExpandedTextAd;
+use Google\AdsApi\AdWords\v201809\cm\Keyword;
+use Google\AdsApi\AdWords\v201809\cm\KeywordMatchType;
+use Google\AdsApi\AdWords\v201809\cm\Money;
+use Google\AdsApi\AdWords\v201809\cm\NetworkSetting;
+use Google\AdsApi\AdWords\v201809\cm\Operator;
+use Google\AdsApi\AdWords\v201809\cm\UrlList;
+use Google\AdsApi\AdWords\v201809\cm\UserStatus;
+use Google\Protobuf\BoolValue;
+use Google\Protobuf\Int64Value;
+use Google\Protobuf\StringValue;
+
+/**
+ * This code example is the second in a series of code examples that shows how to create
+ * a Search campign using the AdWords API, and then migrate it to the Google Ads API one
+ * functionality at a time. See CreateCompleteCampaignAdwordsApiOnly.php through
+ * CreateCompleteCampaignBothApisPhase4.php for code examples in various stages of migration.
+ *
+ * In this code example, the functionality to create a campaign budget
+ * has been migrated to the Google Ads API. The remaining functionalities - creating a campaign,
+ * creating an ad group, creating expanded text ads and creating keywords - are using the AdWords
+ * API.
+ */
 class CreateCompleteCampaignBothApisPhase1
 {
-    public static function runExample()
-    {
-        echo "Not implemented yet";
+
+    // Number of ads being added/updated in this code example.
+    const NUMBER_OF_ADS = 5;
```

Comment:
Could you use private const for all the constants you created?

Reply:
Done.